packages/gallery/src/components/gallery/proGallery/galleryContainerExtraNew.js

```diff
@@ -68,7 +64,7 @@ export class GalleryContainer extends React.Component {
   componentDidMount() {
     this.scrollToItem(this.props.currentIdx, false, true, 0);
     this.eventsListener(EVENTS.APP_LOADED, {});
-    this.videoScrollHelper.initializePlayState();
+    this.videoScrollHelper && this.videoScrollHelper.initializePlayState();
```

Comment:
this looks bad. The videoScrollHelper needs to handle its state.
The videoScrollHelper can have a wrapper of some sort that is always defined